Sources/TokamakDemo/TokamakDemo.swift

```diff
@@ -40,122 +40,108 @@ struct NavItem: Identifiable {
-            Text(link.id)
-            #elseif os(macOS)
-            Text(link.id).opacity(0.5)
+        List {
+          Section(header: Text("Buttons")) {
+            NavItem("Counter", destination:
+              Counter(count: Count(value: 5), limit: 15)
+                .padding()
+                .background(Color(red: 0.9, green: 0.9, blue: 0.9, opacity: 1.0))
+                .border(Color.red, width: 3))
+            NavItem("ButtonStyle", destination: ButtonStyleDemo())
+          }
+          Section(header: Text("Containers")) {
+            NavItem("ForEach", destination: ForEachDemo())
+            #if !os(macOS)
+            if #available(iOS 14.0, *) {
+              NavItem("List", destination: ListDemo().listStyle(InsetGroupedListStyle()))
+            } else {
+              NavItem("List", destination: ListDemo())
+            }
             #else
-            HStack {
-              Text(link.id)
-              Spacer()
-              Text("unavailable").opacity(0.5)
+            NavItem("List", destination: ListDemo())
+            #endif
+
+            if #available(iOS 14.0, *) {
+              NavItem("Sidebar", destination: SidebarListDemo().listStyle(SidebarListStyle()))
+            } else {
+              NavItem(unavailable: "Sidebar")
+            }
+            if #available(OSX 10.16, iOS 14.0, *) {
+              NavItem("OutlineGroup", destination: OutlineGroupDemo())
+            } else {
+              NavItem(unavailable: "OutlineGroup")
+            }
+          }
+          Section(header: Text("Layout")) {
+            if #available(OSX 10.16, iOS 14.0, *) {
+              NavItem("Grid", destination: GridDemo())
+            } else {
+              NavItem(unavailable: "Grid")
             }
+            NavItem("Spacer", destination: SpacerDemo())
+            NavItem("ZStack", destination: ZStack {
+              Text("I'm on bottom")
+              Text("I'm forced to the top")
+                .zIndex(1)
+              Text("I'm on top")
+            }.padding(20))
+          }
+          Section(header: Text("Selectors")) {
+            NavItem("Picker", destination: PickerDemo())
+            NavItem("Toggle", destination: ToggleDemo())
+          }
+          Section(header: Text("Text")) {
+            NavItem("Text", destination: TextDemo())
+            NavItem("TextField", destination: TextFieldDemo())
+          }
+          Section(header: Text("Misc")) {
+            NavItem("Path", destination: PathDemo())
+            NavItem("Environment", destination: EnvironmentDemo().font(.system(size: 8)))
+            NavItem("Color", destination: ColorDemo())
+            if #available(OSX 11.0, iOS 14.0, *) {
+              NavItem("AppStorage", destination: AppStorageDemo())
+            } else {
+              NavItem(unavailable: "AppStorage")
+            }
+            if #available(OSX 11.0, iOS 14.0, *) {
+              NavItem("Redact", destination: RedactDemo())
```

Comment:
I'm still not sure if this should be "Redacted", not "Redact"? But I'm not a native English speaker, so maybe the latter does make more sense in this context?

Reply:
Good point! I went with “Redaction” which I think makes a little more sense.